erpnext/stock/utils.py

```diff
@@ -284,6 +284,10 @@ def is_group_warehouse(warehouse):
 	if frappe.db.get_value("Warehouse", warehouse, "is_group"):
 		frappe.throw(_("Group node warehouse is not allowed to select for transactions"))
 
+def validate_disabled_warehouse(warehouse):
```

Comment:
should apply to stock reconciliation as well right ? Stock Reconciliation's validate doesn't trigger Stock Controller's validate.
So you can manually call this function there too or maybe add in the stock ledger entry validate. Whatever fits you best.

Reply:
Not needed, it will be validated via Stock Ledger Entry